gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -168,6 +170,33 @@ public static ProjectListOption fields(ProjectField... fields) {
   }
 
   /**
+   * The permissions associated with a Google Cloud project. These values can be used when calling
+   * {@link #testPermissions}.
+   */
+  public enum Permission {
+    DELETE("delete"),
```

Comment:
lets verify that this is the list the service wants to expose.
Would be good if the service updates the doc with this list (like other services that uses IAM) and for us to `@see` that page.